dist/hexagon-dark/hexagon.js

```diff
@@ -15257,7 +15256,7 @@ ColorPicker = (function(superClass) {
           return circleMoved(pos);
         };
         dragObject = function(elem, parent, min, max, startCallback, moveCallback, endCallback) {
-          var StartListening, StopListening, cursorStartPos, disposed, dragGo, dragStart, dragStop, dragStopHook, dragging, elementStartPos, listening, temp;
```

Comment:
The `dist/` files don't normally get updated until a release - could these bits be left out the PR?